src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -48,15 +48,31 @@ const AccordionPanel = forwardRef(
           aria-expanded={active}
           onClick={onPanelChange}
           onMouseOver={event => {
-            setHover(theme.dark ? 'light-4' : 'dark-3');
+            setHover(
+              theme.dark
+                ? (theme.accordion.heading.hover &&
+                    theme.accordion.heading.hover.color) ||
+                    'light-4'
+                : (theme.accordion.heading.hover &&
+                    theme.accordion.heading.hover.color) ||
+                    'dark-3',
+            );
             if (onMouseOver) onMouseOver(event);
           }}
           onMouseOut={event => {
             setHover(undefined);
             if (onMouseOut) onMouseOut(event);
           }}
           onFocus={event => {
-            setHover(theme.dark ? 'light-4' : 'dark-3');
+            setHover(
```

Comment:
Setting hover while focus doesn't feel right to me. but I might be missing something.

Reply:
That was the original code. When the accordion is tabbed through the heading title still gets hover behavior I can look more behind this about best practices

Reply:
Im not sure how others would feel if we took out the hover on focus if they are already having it there and since the code has been that way. I did look at some other accordions and how they were being used as well as some accessibility with accordion https://www.w3.org/TR/wai-aria-practices/examples/accordion/accordion.html for instance and saw mostly just the focus not hover but again since this was here before Im open for either way.